test/unit/Raffle.test.js

```diff
@@ -8,198 +8,194 @@ const {
           await network.provider.send("evm_increaseTime", [
             interval.toNumber() + 1,
           ]);
-          await network.provider.send("evm_mine", []);
-          const tx = await raffle.performUpkeep([]);
+          await network.provider.request({ method: "evm_mine", params: [] });
+          const tx = await raffle.performUpkeep("0x");
           assert(tx);
         });
-        it("reverts when checkupkeep is false", async function () {
-          await expect(raffle.performUpkeep([])).to.be.revertedWith(
+        it("reverts if checkup is false", async () => {
+          await expect(raffle.performUpkeep("0x")).to.be.revertedWith(
             "Raffle__UpkeepNotNeeded"
           );
         });
-        it("updates the raffle state, emits event and calls the vrf coordinator", async function () {
+        it("updates the raffle state and emits a requestId", async () => {
           await raffle.enterRaffle({ value: raffleEntranceFee });
           await network.provider.send("evm_increaseTime", [
             interval.toNumber() + 1,
           ]);
-          await network.provider.send("evm_mine", []);
-          const txResponse = await raffle.performUpkeep([]);
+          await network.provider.request({ method: "evm_mine", params: [] });
+          const txResponse = await raffle.performUpkeep("0x");
           const txReceipt = await txResponse.wait(1);
-          const requestId = txReceipt.events[1].args.requestId;
           const raffleState = await raffle.getRaffleState();
+          const requestId = txReceipt.events[1].args.requestId;
           assert(requestId.toNumber() > 0);
-          assert(raffleState.toString() == "1");
+          assert(raffleState == 1);
         });
       });
       describe("fulfillRandomWords", function () {
-        beforeEach(async function () {
+        beforeEach(async () => {
           await raffle.enterRaffle({ value: raffleEntranceFee });
           await network.provider.send("evm_increaseTime", [
             interval.toNumber() + 1,
           ]);
-          await network.provider.send("evm_mine", []);
+          await network.provider.request({ method: "evm_mine", params: [] });
         });
-        it("can only be called after performUpkeep", async function () {
+        it("can only be called after performupkeep", async () => {
           await expect(
             vrfCoordinatorV2Mock.fulfillRandomWords(0, raffle.address)
           ).to.be.revertedWith("nonexistent request");
           await expect(
             vrfCoordinatorV2Mock.fulfillRandomWords(1, raffle.address)
           ).to.be.revertedWith("nonexistent request");
         });
-        it("picks a winner, resets the loterry and sends money", async function () {
-          const additionalEntrants = 3;
-          const startingAccountIndex = 2;
-          const accounts = await ethers.getSigners();
+
+        it("picks a winner, resets, and sends money", async () => {
```

Comment:
Start of relevant changes
TODO: Create a PR that only has the relevant changes to make `yarn hardhat test` command pass.

Reply:
These changes are made in the main branch. The test command passes.